### DISCUSSION TITLE
pgp support?

### DISCUSSION THREAD

**Discussion by braboobssiere** [forwardemail/forwardemail.net#180] [Q&A] [answered]
(I'm not a user of forwardemail yet, so excuse me if this is already existing feature)

does it possible to set forwardemail to auto encrypted email with my public key on forwarding (so that my email provider doesn't get email content)? and is this already existing feature, and/or someone already suggested it?

thank you

**Comment by titanism** [comment]
We can look into this after SMTP and error logs, and a few other priorities.  This isn't currently supported - we don't modify emails.

**Comment by titanism** [comment, marked as answer]
We have added OpenPGP/WKD support to IMAP/POP3/SMTP, see <https://forwardemail.net/en/faq#do-you-support-openpgpmime-end-to-end-encryption-e2ee-and-web-key-directory-wkd>.
